profiler/upload.go

```diff
@@ -23,7 +23,12 @@ import (
 // maxRetries specifies the maximum number of retries to have when an error occurs.
 const maxRetries = 2
 
+var errOldAgent = errors.New("datadog agent is not accepting profiles. Agent-based profiling deployments " +
+	"require Datadog agent >= 7.20")
+
 var httpClient = &http.Client{
+	// TODO(alexjf): Lets bump this? Intake times out at 30 seconds, we probably shouldn't bail out earlier than this.
+	//               This might require some reworking of the upload queue though...
```

Comment:
Should we just make an issue or a ticket out of this? I don't know if readers stopping over it will ponder about changing it.